### PR TITLE
Update dependencies for Sylius 1.9

### PR DESCRIPTION
**Pull request by MacBim**
This follows the previous PR #72 and updates the dependencies needed for Sylius 1.9

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "php": ">=7.4",
         "doctrine/collections": "^1.6",
         "doctrine/orm": "^2.7",
-        "doctrine/persistence": "^1.3",
+        "doctrine/persistence": "^2.1.0",
```
Comment (loevgaard) [inline composer.json:16]:
```suggestion
        "doctrine/persistence": "^1.3 || ^2.1",
```

File: composer.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
         "roave/security-advisories": "dev-master",
         "setono/code-quality-pack": "^1.5.2",
         "setono/sylius-behat-pack": "^0.1",
-        "sylius/sylius": "~1.7.0",
+        "sylius/sylius": "~1.9.0",
```
Comment (loevgaard) [inline composer.json:36]:
```suggestion
        "sylius/sylius": "~1.7.0",
```

Do not edit this :)

**Review by loevgaard — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (MacBim) [reply composer.json:36]: Woops. My bad, will change back !

File: tests/Application/config/bundles.php [hunk 3]
```diff
@@ -40,7 +39,7 @@
     Liip\ImagineBundle\LiipImagineBundle::class => ['all' => true],
     Payum\Bundle\PayumBundle\PayumBundle::class => ['all' => true],
     Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle::class => ['all' => true],
-    WhiteOctober\PagerfantaBundle\WhiteOctoberPagerfantaBundle::class => ['all' => true],
+    BabDev\PagerfantaBundle\BabDevPagerfantaBundle::class => ['all' => true],
```
Comment (loevgaard) [inline tests/Application/config/bundles.php:42]: Do not change this either :)

Comment (MacBim) [reply tests/Application/config/bundles.php]: My mistake. I reverted the modification

**Comment by MacBim** [comment]
If builds succeed, i'll rebase so there is only one commit

**Comment by MacBim** [comment]
@loevgaard It should be all good now !

**Comment by loevgaard** [comment]
Will merge and release now :)

**Pull request merged**